lavamoat/browserify/beta/policy.json

```diff
@@ -2811,6 +2811,22 @@
         "crypto.getRandomValues": true
       }
     },
+    "@metamask/controllers>web3": {
+      "globals": {
+        "Web3": "write",
+        "XMLHttpRequest": "write",
```

Comment:
@amerkadicE @mirjanaKukic: I'm wondering if this may be the issue.
Changing `"XMLHttpRequest": "write",` to `"XMLHttpRequest": true,` seems to fix the tests for me. I'm not sure how it got changed to "write" based on its previous value.

Reply:
Thank you @PeterYinusa . Tests are passing on the pipeline as well with this fix.